images/vep/Dockerfile

```diff
@@ -2,27 +2,43 @@ FROM debian:stretch-slim
 
 ENV MAMBA_ROOT_PREFIX=/root/micromamba
 ENV PATH=$MAMBA_ROOT_PREFIX/bin:$PATH
-ARG VERSION=${VERSION:-104.3}
+ARG VERSION=${VERSION:-105.0}
 
 # hadolint ignore=DL3008
 RUN apt-get update && \
-    apt-get install -y git wget bash bzip2 zip lsb-release curl gnupg2 && \
+    apt-get install --no-install-recommends -y \
+    bash \
+    bzip2 \
+    curl \
+    git \
+    gnupg2 \
+    lsb-release \
+    wget \
+    zip \
```

Comment:
Can't see if it's just reformatted/reodered or a package added here?

Reply:
ah yeah, this was just splitting the packages out and ordering alphabetically - no changes